### PR TITLE
optimize integrate framework

### PR DESCRIPTION
**Pull request by cherrylzhao**
for #5301
- display raw sql in parametrized test
- print test context when assertion error occur

### CODE REVIEW

File: sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/SingleIT.java [hunk 1]
```diff
@@ -39,13 +42,29 @@
...
+        this.sql = convert(sql);
         expectedDataFile = getExpectedDataFile(path, ruleType, databaseType, null != assertion ? assertion.getExpectedDataFile() : null);
     }
+    
+    private String convert(final String sql) throws ParseException {
+        return caseType == SQLCaseType.Literal ? getLiteralSQL(sql) : sql;
+    }
+    
+    private String getLiteralSQL(final String sql) throws ParseException {
+        final List<Object> parameters = null != assertion ? assertion.getSQLValues().stream().map(SQLValue::toString).collect(Collectors.toList()) : null;
```
Comment (terrymanu) [inline sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/SingleIT.java:62]: final maybe unnecessary

File: sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/util/IntegrateTestParameters.java [hunk 2]
```diff
@@ -168,7 +152,7 @@ private static void printTestPlan(final Map<DatabaseType, Collection<Object[]>>
             disabledPlan.add(String.format("%s(%s)", entry.getKey().getName(), entry.getValue().size() * factor));
         }
         System.out.println("[INFO] ======= Test Plan =======");
-        String summary = String.format("[%s] Total: %s, Active: %s, Disabled: %s",
+        String summary = String.format("[%s] Total: %s, Active: %s, Disabled: %s \n",
```
Comment (terrymanu) [inline sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/util/IntegrateTestParameters.java:155]: `\n` should be `LINE_SEPARATOR`

File: sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/SingleIT.java (hunk 1 above)
Comment (cherrylzhao) [reply sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/SingleIT.java]: have fixed it

File: sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/util/IntegrateTestParameters.java (hunk 2 above)
Comment (cherrylzhao) [reply sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/util/IntegrateTestParameters.java]: have fixed it

**Pull request merged**